virtualhosts/etc/e-smith/templates/etc/opt/rh/rh-php72/php-fpm.d/000-virtualhost.conf/10Base

```diff
@@ -0,0 +1,73 @@
+{
+    use esmith::ConfigDB;
+    my $vdb = esmith::ConfigDB->open_ro('vhosts') || die("Can't open vhosts db");
+
+    foreach my $vhost ($vdb->get_all_by_prop('type'=>'vhost')) {
+      
+        my $VhostName = $vhost->key;
+        my $status = $vhost->prop('status') || 'disabled';
+        my $PhpRhVersion = $vhost->prop('PhpRhVersion') ||' default';
+
+        #skip if vhost status is not enabled
+        if ($status ne 'enabled' ){
+          next;
+        }
+        #skip if vhost php version is not php72
+        if ($PhpRhVersion ne 'php72' ){
+          next;
+        }
+
+        my $PhpCustomSettings = $vhost->prop('PhpCustomSettings') || 'disabled';
+        my $ServerNames = $vhost->prop('ServerNames') || '';
+        $ServerNames =~ s/,/ /g;
+
+        $OUT .= qq(
+;
+; vhosts $ServerNames
+;
+
+[$VhostName]
+listen = /var/run/rh-$PhpRhVersion-php-fpm/$VhostName-$PhpRhVersion.sock
+
+;Logs
+php_admin_value[error_log] = /var/opt/rh/rh-$PhpRhVersion/log/php-fpm/error.log
```

Comment:
Is it reasonable to split error log by pool? Is it safe to use `$VhostName` in the error log file name (i.e. old nethgui vhosts have a name set by the user)

```suggestion
php_admin_value[error_log] = /var/opt/rh/rh-$PhpRhVersion/log/php-fpm/error-$VhostName.log
```

Reply:
this is what I did first, but I think it is easier like apache to get all error in one file rather to looks after all, but I can reverse...sometime yo prefer to get all in one file (like configuration for example)

Reply:
I guess there isn't any reference to the pool name. If I have multiple applications separate log files can help to understand the message/issue origin.

Reply:
ok :D

Reply:
done